Tests/MVC6/Boilerplate.AspNetCore.TagHelpers.Test/project.json

```diff
@@ -1,20 +1,23 @@
 {
   "version": "1.0.0-*",
-
   "testRunner": "xunit",
-
+  "title": "",
+  "buildOptions": {
+    "debugType": "portable",
+    "preserveCompilationContext": true,
+    "additionalArguments": [ "/ruleset:../../../MinimumRecommendedRulesWithStyleCop.ruleset" ],
+    "copyToOutput": {"include": [ "xunit.runner.json" ]},
+    "xmlDoc": true
+  },
   "dependencies": {
-    "Boilerplate.AspNetCore.TagHelpers": "*",
+    "Boilerplate.AspNetCore.TagHelpers": "2.1.0",
     "dotnet-test-xunit": "2.2.0-preview2-build1029",
+    "Microsoft.AspNetCore.Razor": "1.1.0",
     "Microsoft.DotNet.InternalAbstractions": "1.0.500-preview2-1-003177",
     "Moq": "4.6.38-alpha",
-    "StyleCop.Analyzers": {
-      "type": "build",
-      "version": "1.0.0"
-    },
+    "StyleCop.Analyzers": "1.0.0",
```

Comment:
For StyleCop, we should always use `"type": "build"`. It matters less for a test project but I've seen strange things happen otherwise.